src/Ontology/Oep4.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Oep4.h"
+#include <HexCoding.h>
+#include <cstdio>
```

Comment:
This header is not required, probably added during a debug. Can you remove it please?

Reply:
I do add this line for debuging. You are right. Thanks